public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (0.46.1).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```

Comment:
>  - Please do NOT serve this file on production.

This looks like an important warning. Did you check that we're following this rule?

Reply:
This is interesting… I made sure MSW is not enabled in production, but technically this file is still served. I'll check this again to see what I can do.